Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "mozjs"
 description = "Rust bindings to the Mozilla SpiderMonkey JavaScript engine."
 repository = "https://github.com/servo/rust-mozjs"
-version = "0.3.2"
+version = "0.3.3"
```

Comment:
This is a breaking change, right ? Should probably be 0.4

Reply:
Yeah, I admit I wanted to cheat a little here :( Wanted to avoid the constant churn with bumping the minor versions, but that should be 0.4, you're right!

Reply:
:eyes: